### ISSUE TITLE
ESLint config no longer working

### ISSUE THREAD

**Issue by svArtist** [Tampermonkey/tampermonkey#1417]
## Expected Behavior
Configs set in Settings->Custom Linter Config are honored

## Actual Behavior
Linter seems unaffected

## Specifications

- Chromium: 98.0.4758.82 (Official Build) (64-bit)
- TM: 4.14
- OS: MS Windows 10 x64

The contents of the settings field, which used to work in previous versions:
```
{
    "env": {
        "browser": true,
        "es2020": true,
        "es6": true,
        "greasemonkey": true,
        "jquery": true
    },
    "extends": "eslint:recommended",
    "globals": {
        "trustedTypes": false
    },
    "parserOptions": {
        "ecmaFeatures": {
            "globalReturn ": true,
            "impliedStrict": true
        },
        "ecmaVersion": 11,
        "sourceType": "script"
    },
    "rules": {
        "curly": "off",
        "no-redeclare": "off",
        "no-unreachable": "off",
        "no-unused-vars": "off",
        "no-implicit-globals": "off"
    }
}```

**Comment by bo33b** [comment]
Same here, since the last update. Very frustrating.
It would seem that the only property value having any effect whatsoever is `"parserOptions": {"ecmaVersion": <number>}` as omitting that does default to 5, meaning eslint quits parsing at the first `const`, `let`, `for...of` etc.

**Comment by derjanb** [comment]
Should be fixed at 4.15.6153 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_4_15_6153.crx))

**Comment by bo33b** [comment]
Confirmed fixed in 4.15, thank you.
